README.md

```diff
@@ -1,2 +1,359 @@
+and traffic allowed into the VM.
+
+The outgoing traffic - e.g. will also match the egress rules - thus allowing
+the VM to communicate to the outside world.
+
+#### Limitations of the Default group
+The Default group cannot be deleted. Never-the-less, its rules can, and new
+rules can also be added to it.
+
+Remember that if you delete the rules within the Default group, the
+connectivity between the VMs whose ports have it attached **will** break.
+
+## Demo scripts
+### Flat scenario
+
+Using the tools provided in this repo, the Default group concept will be
+showcased, and two different configurations will be shown: the first will
+grant ICMP access amongst a set of VMs, the second will grant access to a web
+server running on one VM. No other kinds of L3 traffic will be allowed.
+
+#### ICMP configuration
+The following ASCII diagram portrays the goal for this scenario.
+In it, there are 3 VMs, all of them belonging to the *icmp* security group.
+That group features a single security group rule, allowing ingress ICMP
+protocol traffic to the VMs.
+
+```
+       +---------------+
+       |      nisu     |
+       |  <dynamic_ip> |
+       |     icmp      |
+       +---------------+
+           |       x
+         icmp     web
+           |       x
+       +---------------+       +---------------+
+       |     net1      |-icmp--|    web_client |
+       | 172.20.50.0/24|x-web-x|  <dynamic_ip> |
+       +---------------+       |      icmp     |
+               |               +---------------+
+       +---------------+
+       |   web_server  |
+       |  <dynamic_ip> |
+       |     icmp      |
+       +---------------+
+
+```
+
+To configure the above scenario, create one external logical network - having
+a subnet on top - through oVirt's UI.
+
+Then, create 3 VMs attached to that external logical network. This can also
+be achieved through the UI.
+
+To create the icmp security group, you can use the [create_icmp_security_group](playbooks/create_icmp_security_group.yml) playbook. Make sure your *clouds.yml*
+file is properly configured, since it specifies the authentication URL and
+credentials from the ansible client to your oVirt engine.
+
+```bash
+ansible-playbook -i localhost create_icmp_security_group.yml
+```
+
+Now that the security group - and rule - are created, the logical ports group
+membership has to feature the ICMP group. To do that, it is required to know
+the security group ID of the icmp group.
+Afterwards, use the *update_ports* playbook, as shown in the example below:
+
+```bash
+ansible-playbook -i localhost update_ports.yml --extra-vars="sec_groups=<icmp_security_group_id>"
+```
+
+**NOTE:** to find the *icmp_security_group_id*, use the [list_openstack_entities.py](list_openstack_entities.py) script - or the OpenStack CLI.
+
+That playbook can update a single port, or all the existing ports. When the
+*port_uuid* variable is not specified, the later occurs - e.g. all ports will
```

Comment:
What does "the later occurs"? did you mean "the following occurs"?

Also, i think i.e. is more appropriate here.

Reply:
later as in former, latter, with a typo, to spice things up. 0_o

Removing the typo.